src/modules/ImportImage/Ui.js

```diff
@@ -9,7 +9,7 @@ module.exports = function ImportImageModuleUi(step, ui) {
     // add a file input listener
     var dropZone = '\
     <div class="dropzone import-image-zone" id="' + dropzoneId + '">\
-          <p>\
+          <p id="dropzone-text">\
```

Comment:
I think this change was not needed.

Reply:
Sorry about that, that was one of my testing changes that I forgot to remove before committing. You're correct, that change was not needed. Thanks for pointing that out!